lib/shopify_cli/theme/syncer.rb

```diff
@@ -387,7 +387,11 @@ def backoff_if_near_limit!(used, limit)
       end
 
       def overwrite_json?
-        @overwrite_json
+        theme_created_at_runtime? | @overwrite_json
```

Comment:
Do we have any specific reason to use the binary `|` instead of the logical `||`?

Reply:
nope! just got my ruby syntax mixed up 😅 ill change it to the logical operator so we don't confuse ourselves when we come back to the code loool